### DISCUSSION TITLE
Expedition 7 - Final reward crash

### DISCUSSION THREAD

**Discussion by homerggg** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#16] [General]
Upon "turning in" the last reward of the expedition (Leviathan - Reward being the Leviathan Frigate), the game crashes.

My uneducated guess is that the Leviathan Frigate Id has change and therefore cannot be redeemed ? Any help I can provide ask away.

![image](https://github.com/user-attachments/assets/7861ec3a-492f-4fb3-9b84-7a54e48d36f1)

EXCEPTION_RECORD:  (.exr -1)
ExceptionAddress: 00007ff66e333db3 (NMS!AK::WriteBytesCount::SetCount+0x0000000000559c73)
  ExceptionCode: c0000005 (Access violation)
  ExceptionFlags: 00000000
NumberParameters: 2
  Parameter[0]: 0000000000000000
  Parameter[1]: 0000000000000000
Attempt to read from address 0000000000000000

![image](https://github.com/user-attachments/assets/8015e755-d0f8-4699-9c70-5a88ff064356)

**Comment by homerggg** [comment]
IMPORTANT additional information : The Leviathan Frigate is available in my main savegame so that would not be the cause of the crash.

To be honest, as long as the frigate is available at the Quicksilver Companion desk, I don't really care if the game crashes or not :-)

Maybe just adding a note to the readme file for that expedition would be enough.

**Reply by cwmonkey** [reply]
Looks like it's not just the Ship of the Damned frigate reward that is broken then, interesting.

Like the Ship of the Damned issue, there's not much we can do about it since this is not a mod (think of it more of a clever configuration hack).

@homerggg Are you able to complete the expedition as normal, or do you have to end it by going online and/or changing out the .json file?

**Reply by homerggg** [reply]
@cwmonkey I cannot end the expedition as normal. Going online and reloading that expedition successfully  forced me to end said expedition.

**Reply by cwmonkey** [reply]
Ok thanks for the info. We can't fix the crashing issue, but we can at least make that milestone "optional" like we did for Phase 4 in Adrift so the expedition can be completed as normal.

**Reply by cwmonkey** [reply]
ok, I flipped a couple rewards around and made the phase 5 milestone optional so the expedition can be ended at the Anomaly expedition terminal. Available here: https://cwmonkey.github.io/nms-expeditions/

I'll add it to this repo eventually.

**Reply by cwmonkey** [reply]
Updated the json in this repo as well.

**Comment by cwmonkey** [comment]
Ah crap, Looks like it's actually the season reward, which is configured differently. I guess what I have to do is make the frigate the reward for completing phase 5, make phase 5 optional, and move phase 5's reward to the season's reward. Blegh.

**Comment by cwmonkey** [comment]
HG seems to have fixed this issue for now.
